src/components/Goods/Goods.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import './Goods.scss';
+import { GoodsTypes } from '../../constants/proptypes';
+import Good from '../Good/Good';
+import Button from '../Button/Button';
+import Select from '../Select/Select';
+
+const SELECT_LENGTH = 10;
+
+export default class Goods extends Component {
+  state = {
+    goods: [...this.props.goods],
+    selectValue: 1,
+    selectList: [...Array(SELECT_LENGTH + 1).keys()]
+      .slice(1)
+      .map(item => ({ name: item, value: item })),
+  };
+
+  reverseClick = () => {
+    this.setState(prevState => ({ goods: prevState.goods.reverse() }));
+  };
+
+  alphabeticalClick = () => {
+    this.setState(prevState => ({ goods: prevState.goods.sort() }));
+  };
+
+  resetClick = () => {
```

Comment:
the same with name,
it seems that you're trying to reset click, but actually you need to reset state of items list